osh/cmd_eval.py

```diff
@@ -111,6 +111,9 @@
 RaiseControlFlow = 1 << 1  # eval/source builtins
 Optimize = 1 << 2
 
+# flags for fnmatch
+FnmCasefold = 1 << 4
```

Comment:
I think we should wrap this constant value in Python, which will make the C++ translation easier:

Example here:


https://github.com/oilshell/oil/blob/master/pyext/posixmodule.c#L2566

and then we can do

from libc import FNM_CASEFOLD